module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -489,10 +492,11 @@ protected function getLocationName($locationId)
             if ($locationResponse->isSuccess()) {
                 $location = json_decode($locationResponse->getBody());
                 $locationName = $location->discoveryDisplayName ?? $location->name;
+                $locationCode = $location->code;
             }
         }
 
-        return $locationName;
+        return [$locationName, $locationCode];
```

Comment:
Again, it might be worth considering returning an associative array here to improve readability. In fact, up above, you could just return `$locationMap[$locationId]` directly up on line 484 instead of decomposing it and then putting it back together again.